src/test/java/com/epam/training/center/hw2/ex2/IndexPage2Test.java

```diff
@@ -0,0 +1,112 @@
+package com.epam.training.center.hw2.ex2;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+public class IndexPage2Test extends AbstractPage2{
+
+    protected String URL = "https://jdi-testing.github.io/jdi-light/index.html";
+
+    @Test(priority = 1, description = "open page and assert browser title")
+    public void browserTitleText() {
+        driver.get(URL);
+        Assert.assertEquals(driver.getTitle(), "Home Page");
+    }
+    //input login and password, assert user name
+    @Test(priority = 2)
+    public void inputLoginDataTest() {
+        WebElement userIcon = driver.findElement(By.id("user-icon"));
+        userIcon.click();
+        WebElement loginField = driver.findElement(By.id("name"));
+        loginField.sendKeys("Roman");
+        WebElement passwordField = driver.findElement(By.id("password"));
+        passwordField.sendKeys("Jdi1234");
+        WebElement loginButton = driver.findElement(By.id("login-button"));
+        loginButton.click();
+        String userName = driver.findElement(By.id("user-name")).getText();
+        Assert.assertEquals(userName, "ROMAN IOVLEV");
+    }
+    //click to service button, click different elements button, assert page is open
+    @Test(priority = 3)
+    public void openAnotherPageTest() {
+        WebElement serviceBtn = driver.findElement(By.cssSelector(".m-l8 > li.dropdown > a"));
+        serviceBtn.click();
+        WebElement diffElementBtn = driver.findElement(By.cssSelector(".open > ul > li:nth-child(8) > a"));
```

Comment:
Navigation like a > b > b in locators is bad practice, pls rewrite locators.

Moreover!
I want you to create method like:

`navigateToPageViaLeftMenu(String pageName)`

I want you to make code parametrized